kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/Main.java

```diff
@@ -96,9 +98,11 @@
     jc.addCommand("schema", new SchemaCommand(console));
     jc.addCommand("info", new InfoCommand(console));
     jc.addCommand("show", new ShowRecordsCommand(console));
+    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
     jc.addCommand("csv-schema", new CSVSchemaCommand(console));
     jc.addCommand("csv-import", new CSVImportCommand(console));
-    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
+    jc.addCommand("json-schema", new JSONSchemaCommand(console));
+    jc.addCommand("json-import", new JSONImportCommand(console));
```

Comment:
Possible, yes. We would probably want to implement the same sub-command structure that is used by Main. But this seems like a lot of work for little gain to me. What is the main benefit of switching from `kite-dataset csv-import ...` to `kite-dataset import csv ...`?

We would still want separate commands so we don't end up with a huge list of options that is the union of CSV, InputFormat, and JSON options. That would be a mess and would be more difficult to keep up to date so users know what options can be applied to each command.